src/_includes/banner.html

```diff
@@ -1,9 +1,9 @@
 <div class="banner">
   <p class="banner__text">
-    Check out the
-    <a href="https://medium.com/dartlang/dart-3-2-c8de8fe1b91f">Dart 3.2 blog post</a>!
+    Dart 3.3 is out!
```

Comment:
Personal preference, but consider an adjustment here:

```suggestion
    Dart 3.3 is here!
```